asana/asana.go

```diff
@@ -246,17 +285,27 @@ func (c *Client) request(method string, path string, data interface{}, opt *Filt
 	if data != nil {
 		req.Header.Set("Content-Type", "application/json")
 	}
+	if form != nil {
+		req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
+	}
 	req.Header.Set("User-Agent", c.UserAgent)
-	resp, err := c.client.Do(req)
+	resp, err := c.client.Do(req.WithContext(ctx))
 	if err != nil {
 		return err
 	}
 	defer resp.Body.Close()
+	if resp.StatusCode == http.StatusUnauthorized {
+		return ErrUnauthorized
+	}
 
 	res := &Response{Data: v}
-	err = json.NewDecoder(resp.Body).Decode(res)
+	b, _ := ioutil.ReadAll(resp.Body)
+	fmt.Println("---------------------------")
+	fmt.Println(resp.StatusCode, string(b))
+	fmt.Println("---------------------------")
```

Comment:
Do these 3 `fmt.Println`s belong here? Did you forget to remove them while testing?